Sources/Build/PkgConfig.swift

```diff
@@ -0,0 +1,161 @@
+/*
+ This source file is part of the Swift.org open source project
+ 
+ Copyright 2015 - 2016 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+ 
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors 
+*/
+
+import Utility
+import func POSIX.getenv
+
+enum PkgConfigError: ErrorProtocol {
+    case CouldNotFindConfigFile
+}
+
+struct PkgConfig {
+    static let searchPaths = ["/usr/local/lib/pkgconfig",
+                              "/usr/local/share/pkgconfig",
+                              "/usr/lib/pkgconfig",
+                              "/usr/share/pkgconfig",
+                              // FIXME: These should only be searched for linux?
+                              "/usr/lib/x86_64-linux-gnu/pkgconfig",
+                              "/usr/local/lib/x86_64-linux-gnu/pkgconfig",
+                              ]
+    
+    let name: String
+    let pcFile: String
+    var cFlags = [String]()
+    var libs = [String]()
+    private var parser: PkgConfigParser
+    
+    init(name: String) throws {
+        self.name = name
+        self.pcFile = try PkgConfig.locatePCFile(name: name)
+        parser = PkgConfigParser(pcFile: pcFile)
+    }
```

Comment:
This is a weird API, having a throwing initializer + not-yet-initialized mutable `cFlags` member variables. Why not just move the `load()` into the initializer? Also, I would recommend using `private` where possible non-private structs, I think it makes things more readable.